websockets/constructor/010.html

```diff
@@ -9,11 +9,11 @@
 <script>
 async_test(function(t) {
   var ws = new WebSocket(SCHEME_DOMAIN_PORT+'/handshake_protocol');
-  ws.onerror = ws.onmessage = ws.onclose = t.step_func(function(e) {assert_unreached(e.type)});
+  ws.onerror = ws.onmessage = ws.onclose = t.unreached_func(e.type);
```

Comment:
`e` is now undefined, right? Maybe this could instead do
```
ws.onerror = t.unreached_func('error');
ws.onmessage = t.unreached_func('message');
ws.onclose = t.unreached_func('close');
```